firebase/firebase.json

```diff
@@ -0,0 +1,35 @@
+{
+  "firestore": {
+    "rules": "firestore.rules",
+    "indexes": "firestore.indexes.json",
+    "predeploy": [
+      "npm --prefix \"$PROJECT_DIR\" run firestore-rules-test"
+    ]
+  },
+  "functions": {
+    "source": "src/functions",
+    "predeploy": [
+      "npm --prefix \"$PROJECT_DIR\" run functions-build"
+    ]
+  },
+  "emulators": {
+    "storage": {
```

Comment:
Firebase Storage emulator was launched recently and still have a lot of bugs. I would remove it while it's not stable

Reply:
Also, we don't use Storage right?

Reply:
I've checked their docs and it seems pretty good all around (even `@firebase/rules-unit-testing` received a significant update with these recent changes). Anyways, these are just standard ports, having these here or not won't make any difference - anyone can start a emulator locally if they wanted to, although all scripts in `package.json` only emulate the specific services that we are planning to use, which are `firestore` and `functions`